### PR TITLE
implement the help-page

### PR DESCRIPTION
**Pull request by Mystica52**
# PR description
PR used creating a help page for the use for the asking the support or feedback
implementation of the help page so that it can help the use to raise the issue and feedback

# Description of tasks that were expected to be completed

- [x] design of the help page
- [x] Submitting it should open a ticket in superadmin account
- [x] superadmin menu has a tickets item
- [x] Tickets appear on the dashboard

# How has this been tested?
`/dashbord/support`
- Pull this pr in local machine
- Set **BAKEND_URL** to `https://devpulse-backend-pr-98.onrender.com/`
- Signin and go on `dashboard/support`
- Submit a message a message
- Sign out and signin as **superadmin**
- Go to `dashboard/ inquiries`

#### Screenshoots

`Help page`

Screenshot (83)

`Inquiries`

Screenshot (91)

### CODE REVIEW

**Review by AbdulKhaliq59 — COMMENTED** [review]
I am just getting this error `Continue seeing toastify notification after already sent message`.
I used to resent a message while I am typing that message, I was continue getting notification.
Screenshot 2023-07-08 at 20 02 08
I am not even get access on `open button` when help message received it,
Screenshot 2023-07-08 at 20 03 00
remove some `console.log` on backend if it is possible

**Review by ceelogre — DISMISSED** [review]
When typing in dark mode, one is unable to see the characters.

image

After submitting a ticket, there's no feedback to the user (no modal or any other reaction that they have submitted a ticket).

What are the credentials to login as superadmin? You only mentioned "login as superadmin" in PR description.
image

Upon successful submission , there's a user toast notification

**Review by oliviertech212 — DISMISSED** [review]
clear form after sending

**Review by oliviertech212 — APPROVED** [review]

**Review by ceelogre — CHANGES_REQUESTED** [review]
Add timestamps in human readable format (an hour ago, today, 2 weeks ago, ...)

image

**Comment by ceelogre** [comment]
When you open the notification, it takes you to the rating page (https://metron-devpulse-6yft9s5mz-metron.vercel.app/dashboard/ratings) which is not what we want.

File: src/utils/timePassedFormating.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import moment from 'moment';
```
Comment (ceelogre) [inline src/utils/timePassedFormating.ts]: Moment is no longer maintained. See: https://momentjs.com/docs/#/-project-status/

**Review by ceelogre — DISMISSED** [review]

File: src/queries/helps.queries.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      user {
+        firstName
+        lastName
+        email
+      }
+    }
+  }
+`;
+
+export default GET_HELPS;
```
Comment (ceelogre) [inline src/queries/helps.queries.tsx]: Helps should be renamed to tickets.

**Review by ceelogre — APPROVED** [review]

**Review by ceelogre — APPROVED** [review]

**Review by fabrice-rw — APPROVED** [review]

**Pull request merged**